_modules/sound.md

```diff
@@ -7,9 +7,57 @@ icon: fa-solid fa-volume
 order: 3
 ---
 
+## Sounds on the Y-Badge
 
-## Challenges
+Your Y-Badge has a buzzer built into it. This buzzer is able to play single-note sounds by turning an electromagnet on and off at high speeds. The magnetic field that is produced vibrates a small disk very quickly, creating the sound you hear. We can specify the note the buzzer plays, and how long that note is played for. However, we can't specify the volume of the sound; it's fixed.
+
+<img src="{% link media/buzzer.png %}" width="400" hspace="5%" vspace="10px">
+
+## Functions
+
+The function to play a sound looks like this:
+```c
+speaker_play_note(<note>, <duration>);
+```
+
+* The `<note>` should be a constant value that we've defined for you (for example, NOTE_C4, which would play a middle C). For a list of other note options, explore the `yboard.h` file.
+* The `<duration>` should be the length you want the note to play in _milliseconds_ (a duration of 1000 would play the note for one second).
+
+## Examples
+To play a G5 for 1/4 second, you would write:
+
+```c
+speaker_play_note(NOTE_G5, 250)
+```
+
+To play "Twinkle Twinkle Little Star", you would use the following code:
+
+```c
+speaker_play_note(NOTE_C4, 400);
+speaker_play_note(NOTE_C4, 400);
+speaker_play_note(NOTE_G4, 400);
+speaker_play_note(NOTE_G4, 400);
+speaker_play_note(NOTE_A4, 400);
+speaker_play_note(NOTE_A4, 400);
+speaker_play_note(NOTE_G4, 800);
+speaker_play_note(NOTE_F4, 400);
+speaker_play_note(NOTE_F4, 400);
+speaker_play_note(NOTE_E4, 400);
+speaker_play_note(NOTE_E4, 400);
+speaker_play_note(NOTE_D4, 400);
+speaker_play_note(NOTE_D4, 400);
+speaker_play_note(NOTE_C4, 800);
+```
+
+
+## Exploration
```

Comment:
Maybe we need some exploration to help them understand the function is non-blocking?  Or should we make it blocking by adding a delay?

Reply:
I think it is a blocking function, based on my board's performance. The buzzer is only capable of vibrating the disk at a single frequency at any given time. In fact, it's difficult to get it to play chords. I haven't explored the use of multiple channels (which is beyond the scope of the camp anyways), but I tried messing around with for loops to alternate the note being played very quickly and had mixed results.